HLTrigger/Configuration/python/customizeHLTforCMSSW.py

```diff
@@ -84,6 +84,17 @@ def customiseFor16569(process):
 
     return process
 
+def customiseFor17094(process):
+    for mod in ['hltHbhereco','hltHbherecoMethod2L1EGSeeded','hltHbherecoMethod2L1EGUnseeded','hltHfreco','hltHoreco']:
+        if hasattr(process,mod):
+            getattr(process,mod).timeSigmaSiPM = cms.double(2.5)
+            getattr(process,mod).pedSigmaSiPM = cms.double(0.00065)
+            getattr(process,mod).noiseSiPM = cms.double(1)
+            getattr(process,mod).ts4Max = cms.vdouble(100.,45000.)
+            getattr(process,mod).ts4chi2 = cms.vdouble(15.,15.)
+
+    return process
+
```

Comment:
I see, thanks for the prompt feedback !
so, this PR does not change the behaviour of the HLT menu
but, for my education, could you please let me know how we can switch the HLT configuration to be 2017 aware ?
I think it would be useful to test it w/in the relval production using the 2017 geometry at HLT as well

thanks a lot !